src/Compiler/Build/WebForms.Compiler.Static.targets

```diff
@@ -4,25 +4,46 @@
     <_HasWebFormsError>false</_HasWebFormsError>
   </PropertyGroup>
 
-  <Target Name="WebFormsCompilation" AfterTargets="CoreBuild" BeforeTargets="GetCopyToOutputDirectoryItems">
+  <Target Name="WebFormsCompilation" AfterTargets="CoreCompile" BeforeTargets="GetCopyToOutputDirectoryItems">
     <PropertyGroup>
       <_WebFormsIsDebug>false</_WebFormsIsDebug>
+      <_WebFormsInputDir>$(MSBuildProjectDirectory)</_WebFormsInputDir>
       <_WebFormsIsDebug Condition=" '$(Configuration.ToUpper())' == 'DEBUG' ">true</_WebFormsIsDebug>
-      <_WebFormsTempDir>$(IntermediateOutputPath)webforms_compile/$([System.Guid]::NewGuid())</_WebFormsTempDir>
+      <_WebFormsTempDir>$(IntermediateOutputPath)webforms_compile/</_WebFormsTempDir>
       <_WebFormsTempDir Condition="!$([System.IO.Path]::IsPathRooted($(_WebFormsTempDir)))">$(ProjectDir)$(_WebFormsTempDir)</_WebFormsTempDir>
       <_WebFormsTempDir>$([MSBuild]::NormalizeDirectory($(_WebFormsTempDir)))</_WebFormsTempDir>
       <_WebFormsPagesFileName>webforms.pages.json</_WebFormsPagesFileName>
       <_WebFormsPagesFile>$(_WebFormsTempDir)/$(_WebFormsPagesFileName)</_WebFormsPagesFile>
     </PropertyGroup>
+
+    <ItemGroup>
+      <_WebFormsCompilerInput Include="@(WebFormsFiles)" />
+      <_WebFormsCompilerInput Include="@(WebFormsDesignerFiles)" />
+      <_WebFormsCompilerInput Include="@(WebFormsCodeBehindFiles)" />
+      <_WebFormsCompilerInput Include="@(WebFormsCodeBehindFiles)" />
+      <_WebFormsCompilerInput Include="@(IntermediateAssembly)" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <_WebFormsCompilerOutput Include="$(_WebFormsTempDir)ASP.$([System.String]::Copy('%(WebFormsFiles.RecursiveDir)%(FileName)%(Extension)').Replace('\\', '_').Replace('/', '_')).dll" />
+    </ItemGroup>
```

Comment:
can we do some task type or at least add a comment ? I can't understand what we are doing here

Reply:
Added some in https://github.com/CoreWebForms/CoreWebForms/pull/256